stable/datadog/templates/agent-psp.yaml

```
@@ -0,0 +1,33 @@
+{{- if .Values.agents.podSecurity.podSecurityPolicy.create}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "datadog.fullname" . }}
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: {{ join "," .Values.agents.podSecurity.seccompProfiles | quote }}
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: {{ join "," .Values.agents.podSecurity.apparmorProfiles | quote }}
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: "runtime/default"
+    apparmor.security.beta.kubernetes.io/defaultProfileName: "runtime/default"
+spec:
+  privileged: {{ .Values.agents.podSecurity.privileged }}
+  hostNetwork: {{ .Values.agents.useHostNetwork }}
+  hostPID: {{ .Values.datadog.dogstatsd.useHostPID }}
+  allowedCapabilities: 
+{{ toYaml .Values.agents.podSecurity.capabilites | indent 4 }}
+  volumes:
+{{ toYaml .Values.agents.podSecurity.volumes | indent 4 }}
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
```

Shouldn’t we have the same value as the one we have for OpenShift in the SCC ?
If we need a particular SElinux context on OpenShift, we should need the same for vanilla kubernetes on SElinux-enabled distribution.

```suggestion
  seLinux:
    rule: MustRunAs
    seLinuxOptions:
      user: "system_u"
      role: "system_r"
      type: "spc_t"
      level: "s0"
```